SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -713,25 +713,12 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment()
 		return;
 	}
 
-	if (!IsSnapshotGenerated())
+	if (!SpatialGDKEditorInstance->IsSnapshotGenerated())
```

Comment:
Actually this is only being used here so it doesn't need to be moved to the SpatialGDKEditor module.